src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs

```diff
@@ -260,6 +271,13 @@ internal static IList<MethodInfo> FindSinkConfigurationMethods(IEnumerable<Assem
             return found;
         }
 
+        internal static IList<MethodInfo> FindAuditSinkConfigurationMethods(IEnumerable<Assembly> configurationAssemblies)
+        {
+            var found = FindConfigurationMethods(configurationAssemblies, typeof(LoggerAuditSinkConfiguration));
```

Comment:
All the other methods somehow used `found.Add(GetSurrogateConfigurationMethod<T>` , but I was not sure whether that was needed for Audit ..

Reply:
@tsimbalar `GetSurrogateConfigurationMethod<T>` is for methods mimicking static methods (extensions) and used just to get `MethodInfo` using expressions without reflection, but implementation is overcomplicated and I just realized you can get `MehtodInfo` right out of method name after casting to appropriate delegate type:
`(Func<LoggerFilterConfiguration, ILogEventFilter, LoggerConfiguration>loggerFilterConfiguration.With).MethodInfo`

Reply:
oh I see. I guess it might not be necessary for `AuditTo`

Reply:
I think it'd be needed for `AuditTo.Logger()` .. but this seems like a bit of a fringe case.